### DISCUSSION TITLE
support format code automatically

### DISCUSSION THREAD

**Discussion by pigletfly** [juicedata/juicefs#3652] [General]
**What would you like to be added**:

We should enable autoformat in the editor so that other contributors can maintain consistent code styles when contributing code. I believe most developers have this option enabled.Or is there a specific reason why we don't auto format the code base ?

**Why is this needed**:

**Comment by SandyXSD** [comment]
We use the default `gofmt`. Is there any advanced tool you think is useful and necessary?
